src/Dynamo.All.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.1.32228.430
+# Visual Studio Version 16
```

Comment:
Any reason to downgrade the VS version?

Reply:
I think this was touched by others who have VS2022, my local VS2019 are still referenced in this case which is `16`